pkg/apis/v1alpha1/vpc_types.go

```diff
@@ -0,0 +1,59 @@
+/* Copyright � 2022 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0 */
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+//+kubebuilder:object:root=true
+//+kubebuilder:subresource:status
+//+kubebuilder:storageversion
+
+// VPC is the Schema for the VPC API
+type VPC struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec   VPCSpec   `json:"spec,omitempty"`
+	Status VPCStatus `json:"status,omitempty"`
+}
+
+//+kubebuilder:object:root=true
+
+// VPCList contains a list of VPC
+type VPCList struct {
+	metav1.TypeMeta `json:",inline"`
+	metav1.ListMeta `json:"metadata,omitempty"`
+	Items           []VPC `json:"items"`
+}
+
+// VPCSpec defines VPC configuration
+type VPCSpec struct {
+}
+
+// VPCStatus defines the observed state of VPC
+type VPCStatus struct {
+	Conditions []Condition `json:"conditions"`
+	// NSX VPC Policy API resource path.
+	NSXPolicyAPIPath string `json:"nsxPolicyAPIPath"`
```

Comment:
I remember that the field should be `NSXResourcePath`. Need confirm from @lxiaopei .